common/users/sb-user.ts

```diff
@@ -38,6 +39,8 @@ export interface SelfUser extends SbUser {
   acceptedTermsVersion: number
   /** The last version of the acceptable use policy this user has seen/accepted. */
   acceptedUsePolicyVersion: number
+  /** The language that this user has chosen to use. */
+  language: TranslationLanguage
```

Comment:
I'd be a little careful assuming this matches a language we actually have active. I don't know what i18next is doing, but imo it makes sense to detect a language and save that info even if we don't currently support it, so that if we support it in the future we can auto-switch to a better experience for that person. That implies that the value we save in the DB is just a locale code, not that it is a `TranslationLanguage` (so this should probably be typed as a string and you could have some method for checking/converting it to a `TranslationLanguage` if needed)